bootstrap/index.js

```diff
@@ -17,20 +17,32 @@ const withStatusCode = (execute) => (data, callback) => {
 const cacheOptions = envOptions()
 if (cacheOptions.enabled) logger.info('Cache enabled: ', cacheOptions)
 
+const withMiddleware = async (execute) => {
+  return await withCache(withStatusCode(execute))
+}
+
 // Execution helper async => sync
-const _executeSync = (execute) => {
-  // Add middleware
-  const _execute = withCache(withStatusCode(execute))
+const executeSync = (execute) => {
+  let _execute = undefined
```

Comment:
Variables are undefined by default

Reply:
Yeah, wanted to be explicit here. But I agree, there is no need.